config.yaml

```diff
@@ -1,3 +1,52 @@
 # Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
-options: {}
+options:
+  webserver_workers:
+    type: int
+    description: The number of webserver worker processes for handling requests.
+  webserver_threads:
+    type: int
+    description: >
+      The number of worker threads for handling requests. Run each webserver worker with the specified number of threads.
+  webserver_keepalive:
+    type: int
+    description: The number of seconds for webserver to wait for requests on a Keep-Alive connection.
+  webserver_timeout:
+    type: int
+    description: Webserver workers silent for more than this many seconds are killed and restarted.
+  flask_env:
+    type: string
+    description: What environment the Flask app is running in, by default it's 'production'.
+  flask_debug:
+    type: boolean
+    description: Whether Flask debug mode is enabled.
+  flask_secret_key:
+    type: string
+    description: >
+      A secret key that will be used for securely signing the session cookie and can be used for any other security
+      related needs by your Flask application. This configuration will set the FLASK_SECRET_KEY environment variable.
+      You need to run `app.config.from_prefixed_env()` in your Flask application in order to receive this configuration.
```

Comment:
s/You need to run/Run? Same for the rest of occurrences

Reply:
Updated, thanks!